scala/scala-impl/src/org/jetbrains/plugins/hydra/settings/HydraApplicationSettings.scala

```diff
@@ -42,6 +50,14 @@ class HydraApplicationSettings extends PersistentStateComponent[HydraApplication
     for { (scalaVersion, _) <- artifactPaths.keySet.toArray } yield scalaVersion
   }
 
+  def getHydraRepositoryName: String = hydraRepositoryUrl.getHost
+
+  def setHydraRepositopryUrl(url: String): Unit = hydraRepositoryUrl = parseUrl(url)
+
+  def getHydraRepositoryUrl: URL = hydraRepositoryUrl
+
+  private def parseUrl(url: String) = Try(new URL(if (url.endsWith("/")) url else url.concat("/"))) getOrElse HydraApplicationSettings.DefaultHydraRepositoryUrl
```

Comment:
Please add a comment about the reason behind the URL manipulation - as discussed yesterday.

Reply:
Sorry. I forgot about it.